SU-HSDO/suhumsci
# Fix(STN-906): Bigger text sizes in collections

# READY FOR REVIEW

## Summary
This work adds Collections to classes we were utilizing for the text sizes for small grid sizes within Rows.  This work should make Collections and Private Collections text titles in Vertical Linked Cards match as they were in Rows.

## Need Review By (Date)
11/25

## Urgency
medium

## Steps to Test
1. npm run test
2. Add a page with multiple Collections and a page with a Row with multiple columns of Vertical Linked Cards and the same content in each paragraph component.
3. Compare these two pages and confirm the text sizes for titles stay the same on both components on all screen sizes.
4. Create a Private page with a Private Collection using the same content and compare it with Row, this should match up just as it did with Collections.
5. Swap themes and make sure nothing everything is still accurate.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
